app/modules/notifications/models.py

```diff
@@ -22,7 +22,7 @@ class NotificationType(str, enum.Enum):
 
 # Can send messages out on multiple channels
 class NotificationChannel(str, enum.Enum):
-    rest = 'Rest API'
+    rest = 'restAPI'
```

Comment:
Any reason?

Reply:
The reason is just because we need to use the string in the schema like this:

```python
class NotificationPreferenceSchema(ExtraValidationSchema): 
    class NotificationChannelSchema(ExtraValidationSchema):
        restAPI = marshmallow.fields.Bool()
        email = marshmallow.fields.Bool()
```

The attribute has to be a valid python variable...